eegnb/experiments/visual_n170/n170.py

```diff
@@ -17,107 +17,39 @@
 from eegnb import generate_save_fn
 from eegnb.devices.eeg import EEG
 from eegnb.stimuli import FACE_HOUSE
+from eegnb.experiments.Experiment import Experiment
```

Comment:
having both the file name and the module name the exact same text (`Experiment`) may be common but is not clean. 

I think the experiment base class should be called something like `BaseExperiment' 

Also I think we should import the final experiment classes into the init so that they can be imported more cleanly at the user level. 

So the final use pattern would be something like

```python
from eegnb.experiments import VisualN170
expt = VisualN170()
expt.present()
```

Reply:
A problem with this is that the file VisualN170 is super buried in the folders, I don't mind bringing all the experiments out of their specific folder so this is possible but need to make sure it does not cause any other problems.